vetro/templates/index.html

```diff
@@ -0,0 +1,43 @@
+<!-- wp:template-part {"slug":"header","tagName":"header"} /-->
```

Comment:
Noting that with the templates this theme currently supplies, `index` would never be used.